app/src/main/java/ar/com/wolox/android/example/utils/Extras.kt

```diff
@@ -40,4 +48,12 @@ object Extras {
         const val UID = "uid"
         const val CLIENT = "client"
     }
+
+    //Convierte una fecha pasada como string a Date, en el formato yyyy-MM-dd
+    fun convertToDate(strdate : String) : DateTime? {
+        val dateFormat = DateTimeFormat.forPattern("yyyy-MM-dd")
+
+        return DateTime.parse(strdate,dateFormat)
+
+    }
```

Comment:
You can make this a String extension function:
https://kotlinlang.org/docs/extensions.html